python/pyspark/mllib/linalg/distributed.py

```diff
@@ -151,6 +151,112 @@ def numCols(self):
         """
         return self._java_matrix_wrapper.call("numCols")
 
+    def computeSVD(self, k, computeU=False, rCond=1e-9):
+        """
+        Computes the singular value decomposition of the RowMatrix.
+
+        The given row matrix A of dimension (m X n) is decomposed into U * s * V'T where
+
+        * U: (m X k) (left singular vectors) is a RowMatrix whose columns are the
+             eigenvectors of (A X A')
+        * s: DenseVector consisting of square root of the eigenvalues (singular values)
+             in descending order.
+        * v: (n X k) (right singular vectors) is a Matrix whose columns are the
+             eigenvectors of (A' X A)
+
+        For more specific details on implementation, please refer the scala documentation.
+
+        :param k: Set the number of singular values to keep.
+        :param computeU: Whether of not to compute U. If set to be True, then U is computed
+                         by A * V * s^-1
+        :param rCond: Reciprocal condition number. All singular values smaller than
+                      rCond * s[0] are treated as zero, where s[0] is the largest
+                      singular value.
+        :returns: SingularValueDecomposition object
+
+        >>> data = [(3, 1, 1), (-1, 3, 1)]
+        >>> rm = RowMatrix(sc.parallelize(data))
+        >>> svd_model = rm.computeSVD(2, True)
+        >>> svd_model.U.rows.collect()
+        [DenseVector([-0.7071, 0.7071]), DenseVector([-0.7071, -0.7071])]
+        >>> svd_model.s
+        DenseVector([3.4641, 3.1623])
+        >>> svd_model.V
+        DenseMatrix(3, 2, [-0.4082, -0.8165, -0.4082, 0.8944, -0.4472, 0.0], 0)
+        """
+        j_model = self._java_matrix_wrapper.call("computeSVD", int(k), computeU, float(rCond))
```

Comment:
Converting `computeU` to a boolean explicitly, `bool(computeU)`, could be useful to allow `0` and `1` to be used.  Alternatively, we could check and enforce a boolean value here, otherwise we get a less-useful error message from the Scala side.